config/rbac/role.yaml

```diff
@@ -1487,3 +1500,63 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
I'm not sure, is this really needed? Why the ODH operator would need those permissions?

Reply:
do not see error after removal